pages/tools.en.html

```diff
@@ -57,65 +57,87 @@ <h1>Internationalization tools</h1>
   <p>This page lists various tools related to internationalization that may be helpful. They are not maintained or owned by the W3C, and the links are provided with no guarrantees.</p>
 </section>
 
+<section id="fonts">
+  <h2><a href="#fonts">Fonts</a></h2>
+  <ul>
+        <li>
+          <p><a href="https://r12a.github.io/scripts/fontlist/index.html">Font lister</a><br>
+            Browser-based app. Provides a (not exhaustive) list of fonts, grouped by script, that are available via the Windows and macOS operating systems, as well as Google's Noto fonts and SIL fonts. Note that some of the Windows and macOS fonts have to be downloaded by the user before they can be applied to text.</p></li>
+  </ul>
+</section>
 
-<section>
-  <h2 id="characters"><a href="#characters">Characters</a></h2>
+
+
+<section id="characters">
+  <h2><a href="#characters">Characters & encoding</a></h2>
   <ul>
-        <li><p><a href="/International/charlint/">Charlint</a><br>
-          A Perl script for character normalization.
-        </p>
-        </li>
         <li>
-          <p><a href="https://r12a.github.io/apps/conversion/">Unicode code converter</a><br>
+          <p><a href="https://r12a.github.io/app-conversion/">Unicode code converter</a><br>
             Browser-based app. Converts between characters, Unicode code point numbers, UTF-8 and UTF-16 code units in hex, Numeric Character References (hex and decimal), percent-encoded text, and other character escape formats.</p></li>
         <li>
           <p><a href="https://r12a.github.io/uniview/">UniView</a><br>
             Browser-based app.
 		  UniView is a Unicode character viewer/picker that  displays  characters from a selected range or search pattern, provides information about them, and allows you to assemble strings from them like in a character map.  You can also create customizable lists of characters for pasting into documents. UniView supports the latest version of the Unicode Standard, including the supplementary planes as well as the BMP. Characters are displayed as graphics by default, but you can also display them using an installed font.</p>
         </li>
         <li>
-          <p><a href="https://r12a.github.io/apps/encodings/">Encoding converter</a><br>
+          <p><a href="https://r12a.github.io/app-analysestring/index.html">Analyse string tool</a><br>
```

Comment:
Seems a bit twisted. Perhaps:

```suggestion
          <p><a href="https://r12a.github.io/app-analysestring/index.html">String analysis tool</a><br>
```

Reply:
I discussed it with @r12a, and since this is the name of the tool, he suggested keeping the original name.

Reply:
Well, the name of the app is "Analyse string", so i think @xfq has it right.  However, a possible compromise may be to move the word 'tool' outside the link text.